landing-page/static/css/landing-page.css

```diff
@@ -239,7 +239,7 @@ h4:hover a { visibility: visible}
 /* Fixed table of contents */
 .grid-container {
     display: grid;
-    grid-template-columns: 5fr 1fr;
+    grid-template-columns: var(--grid-dimensions);

```

Comment:
This is so that we can have the grid sections variable, based on if we show the TOC on a particular page or not.